proposals/0024-opacity-micromaps.md

```diff
@@ -193,37 +194,19 @@ shader model 6.9. `RAYQUERY_FLAG_NONE` will be left unrestricted.
 
 This will have implications for uses of these flags outside of the intended
 targets.  Since they are just uint values, it's technically legal to refer to
-them elsewhere, so we will use a warning which defaults to warning rather than
-using DefaultError like we would for calls to unsupported intrinsic functions.
+them elsewhere, so we will use a DefaultError warning, since this will be the
+only diagnostic used to catch use of these flags in an earlier shader model.
 
 Proposed warning diagnostic:
 
 - `"potential misuse of built-in constant %0 in shader model %1; introduced in shader model %2"`.
   This new warning will have a new warning group to allow it to be targeted
   easily for command-line override, such as `hlsl-availability-constant`.
 
-When compiling a library with the
-`RaytracingPipelineFlags::AllowOpacityMicromaps` flag set in a
-[Raytracing pipeline config1][pipeline-config] subobject,
-a new DefaultError warning diagnostic will be added if the shader model is less
-than 6.9. This will detect the case where the flag is not spelled out and
-caught by AST traversal.
-
 Current HLSL diagnostics in DXC do not verify `RayFlags` values in any context.
 `TraceRay()` and `RayQuery::TraceRayInline()` accept non-immediate values, but
 the `RayFlags` provided as a template argument to `RayQuery` must be immediate.
```

Comment:
This paragraph was intended to provide context for the other checks when flags were immediate.  However, it seems a little out of place now.  I mentioned you could probably remove it in our conversation yesterday.